specification.markdown

```diff
@@ -108,7 +106,7 @@ The VINT\_DATA portion of the Variable Size Integer includes all data that follo
 
 ## VINT Examples
 
-This table shows examples of Variable Size Integers with lengths from 1 to 5 octets. The Usable Bits column refers to the number of bits that can be used in the VINT\_DATA. The Representation column depicts a binary expression of Variable Size Integers where VINT\_WIDTH is depicted by `0`, the VINT\_MARKER as `1`, and the VINT\_DATA as `x`.
+[@tableUsableBits] shows examples of Variable Size Integers with lengths from 1 to 5 octets. The Usable Bits column refers to the number of bits that can be used in the VINT\_DATA. The Representation column depicts a binary expression of Variable Size Integers where VINT\_WIDTH is depicted by `0`, the VINT\_MARKER as `1`, and the VINT\_DATA as `x`.
```

Comment:
Good.

It seems Table 5 and Table 15 are not referenced. Not sure it's important.

Reply:
do you suggest captions for them?

Reply:
I added some.